_locales/_untranslated_en.json

```diff
@@ -59,5 +59,11 @@
   },
   "reset_encryption_confirm": {
     "message": "Forget encryption key for this contact."
+  },
+  "voice_send": {
+    "message": "Send a voice message"
+  },
+  "voice_send_cannot": {
```

Comment:
this seems unused?

Reply:
It's used (now) as aria-label for the microphone button